config/defaults.conf

```diff
@@ -71,7 +71,8 @@ pretrain_tasks = sst  // Comma-separated list of pretraining tasks or 'glue' or
                       // do_pretrain to 1, and do_target_task_training to 0.
 target_tasks = glue  // Target tasks, for use in both target_task_training and the final
                      // evaluation, in the same list format as pretrain_tasks.
-
+                     // If do_full_eval = 1, then target_tasks will be used for 
+                     // model evaluation. 
```

Comment:
The previous sentence says the same thing ('for use in the final evaluation'). Merge the two sentences or, preferably, just revert the change. We're going back to the behavior from before last month, so the documentation should mostly be correct.

Reply:
I added the specifics about do_full_eval to make it extra clear that even if do_target_train = 0, target_tasks will be evaluated on.